app/helpers/tv_series_helper.rb

```diff
@@ -0,0 +1,11 @@
+module TvSeriesHelper
+
+  def image_for_tv_poster(series)
```

Comment:
I could not easily reuse the `image_for` helper that we made for movies without a larger refactor. I decided to make a new version for TV instead.